### ISSUE TITLE
Incorrect error in line 0 when CFML parsing encounters cfscript-like not-equals

### ISSUE THREAD

**Issue by mpaluchowski** [cflint/CFLint#230]
When scanning the following piece of code:

```cfml
<cfif "foo" <> "bar">Foo</cfif>
```

The `<>` triggers a parsing error (which might be an issue in itself -- I'm not sure if today's standards allow script-like use of comparators in CFML; **Update:** the code _does_ work correctly on Railo 4.2.), which erroneously appears twice in the report output, once correctly with reported line 1, once incorrectly with line 0:

```shell
$ java -jar CFLint-0.11.1-all.jar -file cfml-test.cfm -text
ERROR StatusLogger No log4j2 configuration file found. Using default configuration: logging only errors to the console.
line 1:7 no viable alternative at input '"foo"<'
Error in: <cfif "foo" <> "bar">Foo</cfif> @ 1:
expression is null, parsing error
Issue
Severity:ERROR
Message code:PARSE_ERROR
        File:C:\Inbox\cflint-test\cfml-test.cfm
        Column:8
        Line:1
                Message:Unable to parse
                Variable:'null' in function:
                Expression:no viable alternative at input '"foo"<'

Severity:ERROR
Message code:PARSE_ERROR
        File:C:\Inbox\cflint-test\cfml-test.cfm
        Column:0
        Line:0
                Message:Unable to parse
                Variable:'null' in function: null
                Expression:
```

**Comment by ryaneberly** [comment]
@mpaluchowski 

This is an interesting parsing ambiguity.

Is the following also valid?

<cfif "foo" > "bar">Foo</cfif>
<cfif "foo" >= "bar">Foo</cfif>

Determining the correctly way to parse that is not trivial....

**Comment by ryaneberly** [comment]
![image](https://cloud.githubusercontent.com/assets/5986747/21599920/2a00ab7e-d142-11e6-9546-eab27f7442a1.png)

**Comment by ryaneberly** [comment]
Lucee 5 seems to parse it though.  Very odd.  I would suggest that trying to use this edgy syntax is a bad idea.

**Comment by mpaluchowski** [comment]
I'm fine if this triggers a `PARSE_ERROR`, but can we have just 1 such error reported and on a valid line? In other words: can we remove the one reported for "line 0"? :) We're feeding these results into SonarQube and this fails now because SonarQube (rightly) reports an invalid line number.

**Comment by ryaneberly** [comment]
That helps.  I agree that the 2nd parsing error should not occur.  it is fixed.
